src/components/HeaderForm.js

```diff
@@ -300,23 +342,35 @@ class HeaderForm extends Component {
     }
   };
   getNextViewTarget = () => ({
+    tracks: new Array(create_track({name: this.state.graphFile, type: fileTypes.GRAPH}), 
+                      create_track({name: this.state.gbwtFile, type: fileTypes.HAPLOTYPE}), 
+                      create_track({name: this.state.gamFile, type: fileTypes.READ}), 
+                      create_track({name: this.state.bedFile, type: fileTypes.BED})),
```

Comment:
Shouldn't we only include the tracks we are actually using?

Also, I'm not sure that the BED files are *really* being used as tracks here; I think they are more like caches, and not BED files we display as regions on the visualization.